client/src/components/Flash/index.js

```diff
@@ -7,14 +7,44 @@ import './flash.css';
 
 function Flash({ flashMessage, onClose }) {
   const { type, message, id } = flashMessage;
+  const [flashMessageHeight, setFlashMessageHeight] = useState(null);
+
+  useEffect(() => {
+    setFlashMessageHeight(
+      document.querySelector('.flash-message').offsetHeight
+    );
+    document.documentElement.style.setProperty(
+      '--flash-message-height',
+      flashMessageHeight + 'px'
+    );
+  }, [flashMessageHeight]);
+
+  function handleClose() {
+    document.documentElement.style.setProperty('--flash-message-height', '0px');
+    onClose();
+  }
```

Comment:
`--flash-message-height` property is used in learn.css and classic.css to balance out the height of the learn page.  Without this, the flex container will be pushed down by the height of the flash message, creating an unneeded scrollbar.